app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TeamsController < ApplicationController
```

Comment:
this file is in the wrong directory. it should be under `app/controllers/api/v1`. also, the class name should be: `Api::V1::TeamsController < ApplicationController`

Reply:
check